README.adoc

```diff
@@ -219,9 +209,9 @@ To collect traces across your systems, you need to implement the OpenTracing `Tr
 interface. For this guide, you can access a bare-bones `Tracer` implementation for
 the Zipkin server in the form of a user feature for Open Liberty.
 
-This feature is already configured for you in your [hotspot file=1]`pom.xml` and [hotspot file=0]`server.xml` files. It will be
-downloaded and installed automatically into each service when you run a Maven build. You can find the [hotspot=zipkinUsr file=0]`opentracingZipkin` feature
-enabled in your [hotspot file=0]`server.xml` file.
+This feature is already configured for you in your [hotspot file=1]`pom.xml` and [hotspot file=0]`server.xml` files. 
+It will be downloaded and installed automatically into each service when you run a Maven build. 
```

Comment:
It will be downloaded and installed automatically
--->
It is automatically downloaded and installed